test/v2_integration.py

```diff
@@ -1661,3 +1661,79 @@ def test_auth_deactivation():
     resp = client.deactivate_authorization(order.authorizations[0])
     if resp.body.status is not messages.STATUS_DEACTIVATED:
         raise Exception("unexpected authorization status")
+
+def get_ocsp_response_and_reason(cert_file, issuer_file, url):
```

Comment:
It looks like this is almost identical to `verify_ocsp` in helpers.py except for the last couple of lines. Could we just update `verify_ocsp` to return the output of `ocsp_verify` (these names were... a bad idea) and avoid having to duplicate this code?

Reply:
Yeah, I didn't do this at first because verification actually fails during the twenty-days-ago setup (openssl isn't pretending to run in the past, so the ocsp response is expired). But it can be made to work with a couple small changes to verify_ocsp. Done.